homeassistant/components/workday/binary_sensor.py

```diff
@@ -6,7 +6,7 @@
 from typing import Any
 
 import holidays
-from holidays import DateLike, HolidayBase
+from holidays.holiday_base import DateLike, HolidayBase
```

Comment:
I can't reproduce the issue on dev branch. I don't see why this should be needed. The `holidays` package does a wildcard import from `holiday_base` which doesn't have `__all__` defined so all names in that module not starting with underscore should be available in `holidays`.

Reply:
I just tried, and I can't reproduce it either, nor does the change make sense from a code perspective.

Going to close this PR for now for those reasons.

../Frenck